patches/minecraft/net/minecraft/client/renderer/entity/layers/LayerArmorBase.java.patch

```diff
@@ -37,11 +37,21 @@
 -                    if (!this.field_177193_i && itemstack.func_77948_v())
 +                }
 +                // Default
-+                    if (!this.field_177193_i && itemstack.func_77962_s())
++                    if (!this.field_177193_i && itemstack.func_77962_s() && net.minecraftforge.client.ForgeHooksClient.renderArmorGlint(itemstack, t, p_177182_1_, p_177182_9_))
                      {
                          this.func_177183_a(p_177182_1_, t, p_177182_2_, p_177182_3_, p_177182_4_, p_177182_5_, p_177182_6_, p_177182_7_, p_177182_8_);
                      }
-@@ -135,11 +134,13 @@
+@@ -114,8 +113,7 @@
+         {
+             GlStateManager.func_179140_f();
+             GlStateManager.func_179112_b(768, 1);
+-            float f2 = 0.76F;
+-            GlStateManager.func_179131_c(0.5F * f2, 0.25F * f2, 0.8F * f2, 1.0F);
++            net.minecraftforge.client.ForgeHooksClient.applyGlintColor();
```

Comment:
Same goes for here, both methods are private and only referenced one time. Modders shouldn't be accessing private methods so it should be safe to break binary compatibility in this instance.